modules/installation-alibaba-config-yaml.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-customizations.adoc
+
+:_content-type: REFERENCE
+[id="installation-alibaba-config-yaml_{context}"]
+= Sample customized `install-config.yaml` file for Alibaba Cloud
+
+You can customize the installation configuration file (`install-config.yaml`) to specify more details about
+your {product-title} cluster's platform or modify the values of the required
```

Comment:
s/your {product-title} cluster's platform/your cluster's platform

Otherwise, this  reads as: "OpenShift Container Platform cluster's platform" :smiley:

Reply:
@ahardin-rh Good catch. It does [read exactly like that](https://docs.openshift.com/container-platform/4.9/installing/installing_aws/installing-aws-customizations.html#installation-aws-config-yaml_installing-aws-customizations)! 
I will fix it here (where I copied it from), too!